static.md

```diff
@@ -20,3 +20,28 @@ module.exports = {
     }]
 };
 ```
+
+If you only want to copy _some_ of the files,
+you can select them using a `filter` function.
+All files passing the test implemented by that function will be copied.
+The function will be called for every file with its path relative to `source`.
+If it returns `true`, the file will be copied,
+otherwise it will not.
+
+In this example,
+we omit images in the `templates/` directory
+and only copy `.ttf` fonts.
+
+```js
+module.exports = {
+    static: [{
+        source: "./images",
+        target: "./public/images",
+        filter: file => !file.startsWith("templates/")
```

Comment:
As a naive user, I just realized that starting with a negated unnecessarily introduces some latent uncertainty. (Not a lot, of course, but Cheating God etc.)

Reply:
I’m not sure whether this comment is informative or appellative in nature.

Reply:
How about passive-aggressive?

Reply:
What do you think of 9f9516d80e5db89bddf8197013d7cf6fa003b42c? Feel free to reject it.

Reply:
Looks good to me